src/Evolution/Systems/GrMhd/ValenciaDivClean/PrimitiveFromConservative.cpp

```diff
@@ -190,6 +190,10 @@ bool PrimitiveFromConservative<OrderedListOfPrimitiveRecoverySchemes,
 using NewmanHamlinThenPalenzuelaEtAl = tmpl::list<
     grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::NewmanHamlin,
     grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::PalenzuelaEtAl>;
+using KastaunThenNewmanThenPalenzuela = tmpl::list<
+    grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::KastaunEtAl,
+    grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::NewmanHamlin,
+    grmhd::ValenciaDivClean::PrimitiveRecoverySchemes::PalenzuelaEtAl>;
```

Comment:
At some point we should explore what parameters each of these fails for and check what the overlaps are... it might be silly to have fallback schemes that are less robust than the primary scheme, as I thought was the case here.

Reply:
agreed, but I can't evolve the blast wave with the Kastaun scheme and I can't evolve a magnetized TOV star with the other schemes :shrug: So to have something work reliably we currently need this, but I agree that the long-term solution is to really understand what is happening.